scripts/aem.js

```diff
@@ -675,6 +631,39 @@ async function waitForLCP(lcpBlocks) {
   });
 }
 
+/**
+ * Loads all blocks in a section.
+ * @param {Element} section The section element
+ */
+
+async function loadSection(section, loadCallback) {
+  const status = section.dataset.sectionStatus;
+  if (status === 'initialized') {
+    section.dataset.sectionStatus = 'loading';
+    const blocks = [...section.querySelectorAll('div.block')];
+    for (let i = 0; i < blocks.length; i += 1) {
```

Comment:
@davidnuescheler this may be counter-intuitive, but if we go this route, would it make sense to actually load the blocks within a section in reverse order.

My thinking:

I've had projects where you end up with nested blocks via auto-blocking. when these blocks are loaded sequentially, the outer block is loaded before the inner block, meaning that the outer block decorate function cannot make decisions based on the content of the inner blocks (because it isn't yet fully formed). There are ways around this on a project basis of course, but it can get messy.

we'd never want to load the blocks for a full page in reverse order as it would delay appearance of top of the page, but since no section gets reveled until all of it's blocks are loaded, maybe within each section it would be safe/performant to do so, allowing nested blocks to be loaded before their `"parent"` blocks, and thus resolving the issue.

(or maybe this is only a problem on a small enough percentage of projects to just not worry about it. feel free to just ignore this)